.github/workflows/integration-tests.yml

```diff
@@ -63,7 +63,6 @@ jobs:
                     APIML_SECURITY_AUTH_JWT_CUSTOMAUTHHEADER: customJwtHeader
                     APIML_SECURITY_AUTH_PASSTICKET_CUSTOMUSERHEADER: customUserHeader
                     APIML_SECURITY_AUTH_PASSTICKET_CUSTOMAUTHHEADER: customPassticketHeader
-                    APIML_SERVICE_CENTRALREGISTRYURLS: https://discovery-service-2:10011/eureka
```

Comment:
this needs to be here for the gateway to register in additional DS, or remove the test givenGatewayIsRegisteredInAdditionalDiscovery_thenReturnInfoFromDiscoveryService

Reply:
it was just a test, restored